src/go/k8s/api/redpanda/v1alpha1/redpanda_types.go

```diff
@@ -12,8 +12,8 @@ package v1alpha1
 import (
 	"encoding/json"
 	"fmt"
+	helmv2beta2 "github.com/fluxcd/helm-controller/api/v2beta2"
```

Comment:
NIT: The norm in operator code is that import group are done in the form

- standard library
- empty line
- external library
- empty line
- internal library